Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -1092,7 +1092,7 @@ static bool getEventTypeFromWebEvent(const WebEvent& event, NSEventType& eventTy
     IntPoint point = frameView->contentsToScreen(IntRect(frameView->windowToContents(event.position()), IntSize())).location();
 
     NSUserInterfaceLayoutDirection uiLayoutDirection = webPage->userInterfaceLayoutDirection() == UserInterfaceLayoutDirection::LTR ? NSUserInterfaceLayoutDirectionLeftToRight : NSUserInterfaceLayoutDirectionRightToLeft;
-    NSMenu *nsMenu = [m_pdfLayerController menuForEvent:nsEventForWebMouseEvent(event) withUserInterfaceLayoutDirection:uiLayoutDirection];
+    RetainPtr<NSMenu> nsMenu = [m_pdfLayerController menuForEvent:nsEventForWebMouseEvent(event) withUserInterfaceLayoutDirection:uiLayoutDirection];
```

Comment:
You should be able to just say `RetainPtr nsMenu = `

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/9c3bdc6957f556075aba0f8d8ec751bd3c4f96fc